php_sql/electivesScript.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE electives (

```

Comment:
You may want to add also the create statement for the DB itself. This way when this script is executed on a new computer it will be totally self-sufficient. You can check for a reference here: https://github.com/aprazova/WEB2020-php/pull/9/files